alphagov/paas-bootstrap
# [Support] Remove bosh CLI gem from Gemfile

## What

We've switched to the v2 bosh cli everywhere, which isn't written in
ruby. We therefore no longer need this here.

Having the bosh gem present interferes with running the tests in paas-cf
as it tends to get picked up on the path before bosh v2.

## How to review

* Verify that my rationale for removing this makes sense
* Verify that Travis passes

## Who can review

Not me.
